receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_metadata.go

```diff
@@ -49,3 +49,13 @@ var cpuTimeDescriptor = func() pdata.MetricDescriptor {
 	descriptor.SetType(pdata.MetricTypeMonotonicDouble)
 	return descriptor
 }()
+
+var cpuUtilizationDescriptor = func() pdata.MetricDescriptor {
+	descriptor := pdata.NewMetricDescriptor()
+	descriptor.InitEmpty()
+	descriptor.SetName("system.cpu.utilization")
+	descriptor.SetDescription("CPU utilization broken down by different states.")
+	descriptor.SetUnit("%")
```

Comment:
Unit is "1"? I did not see "%" in the list of standard units.

Reply:
Where is the list of standard units btw? I could not find this anywhere and had thought 1 meant a "count".

Reply:
You can check in the proto there is a link and in the specs I believe. We also use the same definition as Stackdriver so you can check internally.

Reply:
Cloud Monitoring, OC, & OT protos all reference http://unitsofmeasure.org/ucum.html which I found very hard to follow as it's not clear which column we should be looking at, etc.

If we are reading from the "definition unit" in that doc, then I guess 1 should be used for a [0-1] percentage value, but that seems very non-descriptive to me.

Reply:
Changed to [0-100] values as discussed